### ISSUE TITLE
Issue with message reader and writer

### ISSUE THREAD

**Issue by digoptical22** [Mainflux/mainflux-lite#1526]
hi guys, i have an issue with storage in mainflux this is:
when i run mongodb writer and reader and i use mainflux-cli to create users and i get an errors like this
 
![image](https://user-images.githubusercontent.com/58322053/145660847-20493698-bb76-4aaf-8821-bf9edfd4e649.png)
Please, help me!!!! Thank you!!!!!

**Comment by digoptical22** [comment]
and i have one more error 
when i run docker-compose --env-file docker/.env -f docker/docker-compose.yml up and this is the error
![image](https://user-images.githubusercontent.com/58322053/145665507-ff00e713-a389-4a1d-93e7-0f46c90e6dff.png)
i think this is error 
I don't know what it is, help me!!! thank you

**Comment by digoptical22** [comment]
a few days ago, i asked about mongodb's writer runs and i got it running, when i wanted to access http://localhost:27017. and I have installed mongo db on my machine using this [link]( https://stackoverflow.com/questions/23446876/cannot-access-mongodb-through-browser-it-looks-like-you-are-trying-to-access-m) . and i think it's the fault right???

**Comment by digoptical22** [comment]
![image](https://user-images.githubusercontent.com/58322053/145716682-1821a161-5a3e-4e22-9a63-a3d55bbd71e8.png)
hi guys, i search that issue i have and i found i can connect mongo db writer without install NATS, but when i'm installing NATS and i have error, some one help me please!!!!!!!!!!

**Comment by dborovcanin** [comment]
@digoptical22 **Please respect issue template**. When closing an issue, please explain how you resolved it and what was the root cause, because it might be useful for the community.
What happened o this one and how did you solve it? For testing purposes (and light development), we suggest you use Docker composition to avoid installing and setting up all the infrastructure components locally. Only when you start hard development and get to know the platform really well, it makes sense to start services you are modifying locally.

**Comment by digoptical22** [comment]
Sorry, I didn't see anyone reply, so I closed it. do you know why i am getting such error??? @dusanb94

**Comment by dborovcanin** [comment]
@digoptical22 It usually takes us a couple of days to respond. Again, please respect the issue template. There are some questions in the template that we need to resolve the issue.
What's the status of Auth database service, did it start? It looks like DB did not start properly. Can you try stopping composition, removing all the orphaned containers, and starting over? Please make sure that all the ports are configured properly (that is, that you don't have conflicts if some local service is taking configured ports).

**Comment by digoptical22** [comment]
Let me describe to you the process where I got the error.
When I started with mainflux, I used mainflux's 0.12.1 tag,Docker (version 20.10),Docker compose (version 1.28),Go (version 1.13.3), Protobuf (version 3.6.1). 
And i read from mainflux's documentation to storage, i want to use another database like cassandra, influx, or mongodb, then i found out about running their writer and reader.
When i run writer and reader to be able to send and view data i am not able to send data. 
When i run writer and reader to be able to send and view data i am not able to send data, and i check writer run there is a line saying "failed to connect to nats: dial tcp:i/o timeout", and I continued to follow the mainflux documentation, and combined with a search, I found an article about mongodb writer, along with a simple download of nats with go get. Here is the link to the guide [](https://github.com/MainfluxLabs/mainflux-mongodb-writer)
And so I followed the link until I used the other command line to download mongodb writer and it got the same error as above.
you read and reply me soon @dusanb94 

thank you very much!
